lib/mintel/dashboards/components/panels/backend-service.libsonnet

```diff
@@ -0,0 +1,82 @@
+local layout = import 'components/layout.libsonnet';
+local commonPanels = import 'components/panels/common.libsonnet';
+{
+
+  overview(serviceSelectorKey='service', serviceSelectorValue='$service', startRow=1000)::
+    local config = {
+      serviceSelectorKey: serviceSelectorKey,
+      serviceSelectorValue: serviceSelectorValue,
+    };
+    layout.grid([
+
+      commonPanels.timeseries(
+        title='Workloads Status',
+        description='Percentage of instances up (by workload)',
+        span=4,
+        max=100,
+        format='percent',
+        legend_show=false,
+        // Fix legendFormat to display deployment/statefulset (needs relabel)
+        thresholds=[
+          {'value': 50,
```

Comment:
is it right to hardcode this here ?

Reply:
No it's not - this block actually needs re-factoring as it's in 3 components right now.

Since it will impact other products, I can do it in a separate MR?

Reply:
Updated / refactored into separate workload panel.